GVFS/GVFS.FunctionalTests/Tests/GitCommands/StatusTests.cs

```diff
@@ -43,15 +45,37 @@ public void CreateFileWithoutClose()
         {
             string srcPath = @"CreateFileWithoutClose.md";
             this.CreateFileWithoutClose(srcPath);
-            this.ValidateGitCommand("status");
+            this.Enlistment.WaitForBackgroundOperations();
+            GVFSHelpers.ModifiedPathsShouldContain(this.Enlistment, this.FileSystem, srcPath);
+            try
+            {
+                this.ValidateGitCommand("status");
+            }
+            catch (Exception ex)
+            {
+                Thread.Sleep(1000);
+                this.ValidateGitCommand("status");
+                Assert.Fail("{0} was succesful on the second try, but failed on first: {1}", nameof(this.ValidateGitCommand), ex.Message);
+             }

```

Comment:
Helper method since this code matches exactly in both tests?

Reply:
I was waiting for someone to suggest that! 

I didn't do it this time since it's only 3 lines of code and I wanted to keep it very explicit.  If it gets used in a third place definitely helper method time.  If I did a helper method it would be private and placed elsewhere in the file.

Reply:
I was thinking all 12 lines of code but up to you.

Reply:
@kewillford, fair point :-).  I added your suggestion.